### PR TITLE
feat(loadModule): optional cache bust key

### PR DESCRIPTION
**Pull request by JAdshead**
- [x]  include holocron update

### CODE REVIEW

**Comment by anescobar1991** [comment]
You should also open a PR to `one-app-dev-cdn` to stop adding the key there. That way the `not-used-in-development` key is omitted since it is no longer needed.

File: src/server/middleware/sendHtml.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ export function renderModuleScripts({
     const { integrity, url: src } = moduleMap.modules[moduleName][bundle];
     const { key } = moduleMap;
     const additionalAttributes = isDevelopmentEnv ? '' : `integrity="${integrity}"`;
-    const scriptSource = isDevelopmentEnv ? src : `${src}?key=${key}`;
+    const scriptSource = isDevelopmentEnv || !key ? src : `${src}?key=${key}`;
```
Comment (jsmartfo) [inline src/server/middleware/sendHtml.js]: are we naming it `key`? 

@PixnBits do your thing ...

Comment (JAdshead) [reply src/server/middleware/sendHtml.js]: This is a discussion between @PixnBits and @10xLaCroixDrinker

**Comment by JAdshead** [comment]
> Warnings
> ⚠️	
> Changes were made to package.json, but not to package-lock.json - _Perhaps you need to run `npm install`?_
> 
> ### 📊 Bundle Size Report
> file name	size on disk	gzip
> app.js	112.7KB	31.4KB
> runtime.js	15KB	5.3KB
> vendors.js	128.1KB	37.9KB
> app~vendors.js	403.5KB	105.2KB
> legacy/app.js	119.4KB	33KB
> legacy/runtime.js	15KB	5.3KB
> legacy/vendors.js	163KB	44.8KB
> legacy/app~vendors.js	409.8KB	106.9KB
> Generated by 🚫 [dangerJS](https://danger.systems/js) against [25a547e](https://github.com/americanexpress/one-app/commit/25a547eea793127688b9a9a162937f87ec2c1da2)

package-lock already container v1.0.0 for holocron

**Pull request merged**